### DISCUSSION TITLE
Grapesjs CLI -   Issue with grapesjs-cli - Stops in the initiation

### DISCUSSION THREAD

**Discussion by jcenlo** [GrapesJS/grapesjs#4889] [Q&A] [answered]
Hi,

I'm having a problem with  "grapesjs-cli": "^4.0.0" it's not working properly and stopping in the initiation. I've tried to initiate the project with the following command:

```
x@MacBook-Pro-de-x grapesjs-my-plugin % npx grapesjs-cli init

Init the project...

? Name of the project (Grapesjs My Plugin) 
x@MacBook-Pro-de-x grapesjs-my-plugin % npx grapesjs-cli init

Init the project...

? Name of the project (Grapesjs My Plugin) 
x@MacBook-Pro-de-x grapesjs-my-plugin % 
```
I would appreciate any help or suggestions in solving this issue. I've tried the following solutions but they didn't work:

Uninstalling and reinstalling the cli with npm uninstall -g grapesjs-cli and npm install -g grapesjs-cli
Initializing the project in a different folder or removing the existing package.json file
Updating to the latest version of Node.js
If anyone has faced a similar issue or has any suggestions, please let me know.

Thanks in advance!

**Comment by artf** [comment]
* Which node version did you try?
* What errors are shown?
* Do you face same issues with the versions prior to v4?
* Did you try to follow the steps as described [here](https://github.com/GrapesJS/cli#plugin-from-0-to-100) (eg. local install not a global one)?

**Comment by jcenlo** [comment]
Hi @artf ,i don't get any error in the console

- Which node version did you try?
v18.13.0 and i also try in node 18.12.4 and no works

- Do you face same issues with the versions prior to v4?
V3 works perfect

- Did you try to follow the steps as described [here](https://github.com/GrapesJS/cli#plugin-from-0-to-100) (eg. local install not a global one)?
yes and still not wroks

- What errors are shown?
no errors in console,this is my npm log


```
0 verbose cli /usr/local/bin/node /usr/local/lib/node_modules/npm/bin/npm-cli.js
1 info using npm@9.4.1
2 info using node@v18.13.0
3 timing npm:load:whichnode Completed in 1ms
4 timing config:load:defaults Completed in 2ms
5 timing config:load:file:/usr/local/lib/node_modules/npm/npmrc Completed in 1ms
6 timing config:load:builtin Completed in 1ms
7 timing config:load:cli Completed in 4ms
8 timing config:load:env Completed in 0ms
9 timing config:load:file:/Users/x/DEV/XditorProject/plugins/booXditor/.npmrc Completed in 0ms
10 timing config:load:project Completed in 5ms
11 timing config:load:file:/Users//.npmrc Completed in 0ms
12 timing config:load:user Completed in 0ms
13 timing config:load:file:/usr/local/etc/npmrc Completed in 0ms
14 timing config:load:global Completed in 0ms
15 timing config:load:setEnvs Completed in 1ms
16 timing config:load Completed in 13ms
17 timing npm:load:configload Completed in 13ms
18 timing npm:load:mkdirpcache Completed in 0ms
19 timing npm:load:mkdirplogs Completed in 1ms
20 verbose title npm exec grapesjs-cli init
21 verbose argv "exec" "--" "grapesjs-cli" "init"
22 timing npm:load:setTitle Completed in 18ms
23 timing config:load:flatten Completed in 2ms
24 timing npm:load:display Completed in 4ms
25 verbose logfile logs-max:10 dir:/Users/x/.npm/_logs/2023-02-02T14_02_46_858Z-
26 verbose logfile /Users/x/.npm/_logs/2023-02-02T14_02_46_858Z-debug-0.log
27 timing npm:load:logFile Completed in 3ms
28 timing npm:load:timers Completed in 1ms
29 timing npm:load:configScope Completed in 0ms
30 timing npm:load Completed in 41ms
31 silly logfile done cleaning log files

```

**Comment by artf** [comment, marked as answer]
Try with this new release https://github.com/GrapesJS/cli/releases/tag/v4.0.1

**Comment by jcenlo** [comment]
Thanks Sensei, you fix bugs like this

![This is an image](https://media1.giphy.com/media/uVQiKrZG7RQ7jteaJI/giphy.gif)
